.github/workflows/issues.yaml

```diff
@@ -4,30 +4,55 @@ name: Create JIRA ticket for new issues
 on:
   issues:
     types: [opened]
-
+permissions: 
+  issues: write
+  contents: read
 jobs:
   jira_task:
     name: Create Jira issue
     runs-on: ubuntu-latest
     steps:
-    - name: Login
-      uses: atlassian/gajira-login@v3.0.1
-      env:
-        JIRA_BASE_URL: ${{ secrets.JIRA_BASE_URL }}
-        JIRA_USER_EMAIL: ${{ secrets.JIRA_USER_EMAIL }}
-        JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
     - name: Create JIRA ticket
-      uses: atlassian/gajira-create@v3.0.1
       id: create
-      with:
-        project: INTMDB
-        issuetype: Task
-        summary: «[HELP] - ${{ github.event.issue.title }}»
-        description: ${{ github.event.issue.html_url }}
-        fields: '{"components": "CloudFormation-PI"}'
+      shell: bash
+      env:
+        ISSUE_TITLE: ${{ github.event.issue.title }}
+        ISSUE_BODY: ${{ github.event.issue.body }}
+        ISSUE_URL: ${{ github.event.issue.html_url }}
+        JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
+      run: |
+        json_response=$(curl --request POST \
+          --url 'https://jira.mongodb.org/rest/api/2/issue' \
+          --header 'Authorization: Bearer '"${JIRA_API_TOKEN}" \
+          --header 'Accept: application/json' \
+          --header 'Content-Type: application/json' \
+          --data '{
+            "fields": {
+                "project": {
+                    "id": "19383"
+                },
+                "summary": "HELP: '"${ISSUE_TITLE}"'",
+                "issuetype": {
+                    "id": "3"
+                },
+                "description": "This ticket tracks the following GitHub issue: '"${ISSUE_URL}"'\n\n Description:\n'"${ISSUE_BODY}"'",
+                "components": [
+                    {
+                        "id": "32225"
+                    }
+                ],
+                "assignee": {
+                  "name": "zuhair.ahmed@mongodb.com"
```

Comment:
can we assign this instead to atlas-devops-integrations@mongodb.com? this is our team email alias so can respond faster to customer request in case I am OOTO

Reply:
sure